src/components/Header/index.scss

```diff
@@ -1,20 +1,13 @@
 .easi-header {
-  .navbar--container {
-    display: none;
-  }
+  // height: 5rem;
```

Comment:
What's this doing?

Reply:
It's doing nothing. I forgot to delete it. On it now!